migrations/20230205214959_update_apps_github_pull_request_table.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+ALTER TABLE apps_github_pull_request
+	ADD github_pull_request_number integer,
+    ADD github_pull_request_url VARCHAR(2048),
+    ADD github_repository_name VARCHAR(20),
+    ADD github_repository_owner VARCHAR(20);
```

Comment:
```suggestion
    ADD github_repository_owner VARCHAR(39);
    ADD github_repository_name VARCHAR(100),
    ADD github_pull_request_number INTEGER,
    ADD github_pull_request_url VARCHAR(2048),
```
Let's set length of `repo name` and `repo owner` based on Github's limits.
<img width="451" alt="image" src="https://user-images.githubusercontent.com/3537801/216904634-8a357102-5bc9-4f21-bae6-254b1e56b3f9.png">

<img width="508" alt="image" src="https://user-images.githubusercontent.com/3537801/216905491-c3867360-3b67-4538-b4ca-d8f2fbecd4f0.png">
<img width="659" alt="image" src="https://user-images.githubusercontent.com/3537801/216905552-29bb608e-98c8-4849-b310-b66a69574c82.png">